q2_types/sample_data/tests/test_transformer.py

```diff
@@ -26,10 +26,9 @@ def test_pd_series_to_alpha_diversity_format(self):
                         name='shannon', index=exp_index)
 
         obs = transformer(exp)
-        '''
-        Squeeze equals true in function call below to convert single column
-        dataframe to series
-        '''
+
+        # Squeeze equals true in function call below to return series instead
```

Comment:
Looking good! What do you think about dropping the "in function call below" portion?